modules/isIterable.js

```diff
@@ -0,0 +1,7 @@
+function isIterable(obj) {
```

Comment:
If you intended `isIterable` to be a public function (i.e., mixed into the `_` object), then you should list it in `modules/index.js`. There should also be a comment before the function to explain its purpose. Take a look at a few other public modules to get a taste of how we write these comments.

If you intended `isIterable` to be only an internal function instead, then you should start the name of the module with a leading underscore, i.e., `modules/_isIterable.js`. In this case it would still be desirable to have a comment that explains the purpose of the function, but I'll readily admit that there are currently more internal functions where this is missing.